packages/openapi-v3/test/unit/controller-spec/controller-decorators/request-body-decorators/primative-types.test.ts

```diff
@@ -0,0 +1,78 @@
+// Copyright IBM Corp. 2018. All Rights Reserved.
+// Node module: @loopback/openapi-v3
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {
+  ControllerSpec,
+  post,
+  requestBody,
+  getControllerSpec,
+} from '../../../../../';
+import {ContentObject, SchemaObject} from '@loopback/openapi-v3-types';
+import {Class} from '@loopback/repository';
+import {expect} from '@loopback/testlab';
+
+// I will add all shortcut decorators if we think they are valuable
+// considering we are changing to DI soon
+describe('requestBody decorator', () => {
+  context('infers request body with primative type - ', () => {
+    let actualSpec: ControllerSpec;
+    let expectedContent: ContentObject;
+
+    it('number', () => {
```

Comment:
"it number" does not read like an English sentence to me.

My proposal:

```ts
describe('requestBody decorator', () => {
  context('for a primitive type', () => {
    it('infers number', () => {
      // ...
    });
  });
});
```

Reply:
@bajtos I think mocha concats the description in `context` and `it`, so the test title will be displayed as:
```
infers request body with primative type - number
```

Reply:
Yes, mocha concats the descriptions. However, the individual lines of code defining a test case should read like an English sentence too! At least that's my ideal.